Makefile

```diff
@@ -14,32 +20,32 @@ build-traffic-generator:
 
 openshift-deploy-kiali-test-depth:
 	@echo About to deploy Kiali Test Depth to OpenShift
-	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '{"meshes": ["kiali-test-depth"]}' -v
+	ansible-playbook ${PLAYBOOK} -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e number_of_namespaces=${NUM_NAMESPACES} -e '{"meshes": ["kiali-test-depth"]}' -v
 
 openshift-deploy-kiali-test-breadth:
 	@echo About to deploy Kiali Test breadth to OpenShift
-	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '{"meshes": ["kiali-test-breadth"]}' -v
+	ansible-playbook ${PLAYBOOK} -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e number_of_namespaces=${NUM_NAMESPACES} -e '{"meshes": ["kiali-test-breadth"]}' -v
 
 openshift-deploy-kiali-test-circle:
 	@echo About to deploy Kiali Test Circle to OpenShift
-	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '{"meshes": ["kiali-test-circle"]}' -v
+	ansible-playbook ${PLAYBOOK} -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e number_of_namespaces=${NUM_NAMESPACES} -e '{"meshes": ["kiali-test-circle"]}' -v
 
 openshift-deploy-kiali-test-circle-callback:
 	@echo About to deploy Kiali Test Circle Callback to OpenShift
-	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '{"meshes": ["kiali-test-circle-callback"]}' -v
+	ansible-playbook ${PLAYBOOK} -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e number_of_namespaces=${NUM_NAMESPACES} -e '{"meshes": ["kiali-test-circle-callback"]}' -v
 
 openshift-deploy-kiali-test-hourglass:
 	@echo About to deploy Kiali Test Hourglass to OpenShift
-	ansible-playbook ./test-service/deploy/ansible/deploy_test_meshes.yml -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e '{"meshes": ["kiali-test-hourglass"]}' -v
+	ansible-playbook ${PLAYBOOK} -e number_of_services=${NUM_SERVICES} -e number_of_versions=${NUM_VERSIONS} -e number_of_namespaces=${NUM_NAMESPACES} -e '{"meshes": ["kiali-test-hourglass"]}' -v
```

Comment:
I think I forgot hourglass to multiple_namespaces

Reply:
@gbaufake do you want to fix that first and then merge this PR after? Or merge this PR now and add hourglass later? Up to you. Just tell me what you want to merge and when to merge it.

Reply:
I think we can merge it with a comment ... then  I submit a PR for that

Reply:
@gbaufake I added a comment - just remove that comment from the makefile when you submit the PR to implement the hourglass stuff.